### ISSUE TITLE
Training steps needed to reproduce the reported number

### ISSUE THREAD

**Issue by dddraxxx** [Visual-Agent/DeepEyes#46]
Thank you for releasing **DeepEyes**! It is an excellent work and means a lot to the community!

I’m now attempting to reproduce the results reported in the paper, and although [here](https://github.com/Visual-Agent/DeepEyes/issues/37#issue-3126619471) suggests 80 steps in paper, I found using your script (7b version) cannot reproduce the results in the paper. (-3% in Vstar and -10%+ in HR bench) Could you clarify the exact steps used to obtain the published numbers? Thanks so much!

**Comment by JaaackHongggg** [comment]
Hi, the RL training is unstable and the 80 step is a suggested step. The optimal steps depend on the number of GPUs, the version of vllm, and some other implement details such as batchsize. 

You can check the reward curve to see if it has converged or is overfitting. Additionally, you can choose to test using the step with the best reward instead of using step 80.

**Comment by dddraxxx** [comment]
Thanks. I try the same config (same batch size and mini batch size) so should be under your standard config. 80 steps seems not enough. Thank you again for your answer.
